week5_sub_strings/spec/spec_helper.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
```

Comment:
i have written about commented lines in the channel

Reply:
removed commented lines